tests/utils.go

```diff
@@ -3377,6 +3377,12 @@ func CreateISCSITargetPOD(containerDiskName cd.ContainerDisk) string {
 	pod, err = virtClient.CoreV1().Pods(NamespaceTestDefault).Get(pod.Name, metav1.GetOptions{})
 	Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("should sucessfully get pod %s after phase changed to Running", pod.Name))
 
+	// DEBUG: print ISCSI target pod IP addresses
```

Comment:
I'm assuming you'll get rid of this commit once you remove the `DMN` tag from this PR.